README.md

```diff
@@ -1 +1,150 @@
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Mobile Frist Portfolio] <a name="about-project"></a>
+
+**[Mobile Frist Portfolio]** is a small one page project that will assist me in getting familiar with the essential tools that I will need to learn and apply in this project and name it as the Set up of Mobile Frist Portfolio.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+**[A header with two parts, the header-top and header-bottom, and some features under]**
+**[under the header-top there is a feature nav that holds the logo,a pop-up link in a feature set for now as an image, and a different div with a content of personal name and general professional description]**
+**[Under the header-bottom a link of different social medias for the time being I set only their icons]**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/ethred/Portfolio.git)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps. Clone this repo using this link: 
+https://github.com/ethred/Portfolio.git
+
+### Prerequisites
+
+In order to run this project you need:
+Web browser and code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+cd my-folder
+git clone https://github.com/ethred/Portfolio.git
+
+### Install
+
+Install this project with: 
+
+
+### Usage
+
+To run the project, execute the following:
+1. Open the terminal/command prompt and navigate to the directory where the "index.html" file is saved; ```cd /path/to/directory```
+2. Once you're in the directory, use the `open` command (on a Mac) or `start` command (on Windows) followed by the name of the HTML file to open it in the default web browser. 
+example; ```open index.html```   or ```start index.html```
+
+
+### Deployment
+
+You can deploy this project using:
+Web Browser (Chrome, edge, firefox...etc)
+Live Preview on Visual Studio Code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@ethred](https://github.com/@ethred/Portfolio.git)
```

Comment:
- [ ] The link to you GitHub profile is still not working ⚠ It is redirecting to Page not found😓 You can use the following to fix this 👇✨

  `[@ethred](https://github.com/ethred)`